hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/server/TopicMetadataLoadingJob.java

```diff
@@ -13,8 +11,6 @@
 
 public class TopicMetadataLoadingJob implements Runnable {
```

Comment:
not really related to this PR but I think it would be helpful to at least add a comment what is this the purpose of this class - i.e. to keep producers warm. Without that it is hard to tell, it refreshes topics in the background and discards the result - what for?